pkg/api/openshiftcluster.go

```diff
@@ -27,6 +27,11 @@ type OpenShiftCluster struct {
 	Lock sync.Mutex `json:"-"`
 }
 
+// IsWimi checks whether a cluster is a Workload Identity cluster or Service Principal cluster
```

Comment:
```suggestion
// IsWorkloadIdentity checks whether a cluster is a Workload Identity cluster or Service Principal cluster
```

Reply:
Ideally, turn on `golangci-lint` and get this type of review automated!